src/applications/gi/tests/e2e/06-vet-tec-mobile.cypress.spec.js

```diff
@@ -0,0 +1,45 @@
+import { initMockProfile, expectLocation } from './cypress-helpers';
+import vetTecProfile from '../data/vet-tec-profile.json';
+import vetTecSearchResults from '../data/vet-tec-search-results.json';
+import { createId } from '../../utils/helpers';
+import { typeOfInstitution, search, selectSearchResult } from './gi-helpers';
+
+describe('VETTEC', () => {
+  beforeEach(() => {
+    cy.route('/v0/gi/institution_programs/search', vetTecSearchResults);
+
+    initMockProfile(vetTecProfile);
+
+    cy.visit('/gi-bill-comparison-tool').injectAxe();
+    cy.axeCheck();
+  });
+
+  it('Default VETTEC profile flow with giBillChapter chapter 33', () => {
+    cy.viewport(481, 750);
+    // Landing Page
+    typeOfInstitution('vettec');
+    cy.axeCheck();
+
+    search();
+    cy.axeCheck();
+    expectLocation('/program-search');
```

Comment:
with the version of the `search()` in this PR this `expectLocation` is not needed

Reply:
removed